locales/de-DE.json

```diff
@@ -116,16 +116,16 @@
   },
   "error": {
     "account_not_found": "Account {0} nicht gefunden",
-    "explore-list-empty": "Momentan ist nichts im Trend. Schau später nochmal vorbei!",
+    "explore-list-empty": "Momentan ist nichts in den Trends. Schaue einfach später noch einmal vorbei!",
     "file_size_cannot_exceed_n_mb": "Die Dateigröße darf {0} MB nicht überschreiten",
-    "sign_in_error": "Kann nicht zu Server verbinden",
+    "sign_in_error": "Kann nicht zu dem Server verbinden",
```

Comment:
```suggestion
    "sign_in_error": "Kann nicht zum Server verbinden",
```

oder

```suggestion
    "sign_in_error": "Kann keine Verbindung zum Server herstellen",
```

Reply:
How about "Kann nicht zum Server verbinden"? Simple and sounds right to me.

Reply:
I'd prefer "...Verbindung herstellen zu..." or "...mit Server verbinden..."

but "...zu Server verbinden.." sounds weird to me